### PR TITLE
average in csharp added

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** None

**Changes:** Added average.cs

### CODE REVIEW

File: code/unclassified/src/average/average.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    class Program
+    {
+        public static void Main(string[] argv)
+        {
+            float sum = 0.0f;
+            float n;
+            foreach(var num in argv)
+            {
+                n = float.Parse(num);
+                sum += n;
```
Comment (arnavb) [inline code/unclassified/src/average/average.cs]: Either move `float n` inside the loop or as another option do `sum += float.Parse(num);` (Unless you want to do error handling, which you should).

File: code/unclassified/src/average/average.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace Average
+{
+    class Program
+    {
+        public static void Main(string[] argv)
+        {
+            float sum = 0.0f;
+            float n;
+            foreach(var num in argv)
```
Comment (arnavb) [inline code/unclassified/src/average/average.cs]: MSDN recommends: `foreach (string num in argv)`.

**Comment by tbhaxor** [comment]
@arnavb kindly review it :smile:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus

**Pull request merged**